packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.scss

```diff
@@ -1,5 +1,5 @@
 .floatingToolbarFill {
-	background-color: $dark-gray-500;
+	background-color: rgba(29, 35, 39, 0.85);
```

Comment:
I think you can use `rgba(#1d2327, 0.85)` here so it's easier to search for that color if we end up making it into a constant in the future